charts/sscs-tribunals-api/values.ccd.preview.template.yaml

```diff
@@ -274,7 +259,7 @@ ccd:
       01_input.conf: |
         input  {
           jdbc {
-            jdbc_connection_string => "jdbc:postgresql://${SERVICE_NAME}-postgresql:5432/data-store?ssl=disable&stringtype=unspecified"
+            jdbc_connection_string => "jdbc:postgresql://${SERVICE_NAME}-postgresql:5432/data-store?sslmode=require&stringtype=unspecified"
```

Comment:
this doesn't look right

Reply:
logstash: has been disabled in the CCD chart  so changing here wouldn't have got tested so will change it back..